apiary.apib

```diff
@@ -1702,8 +1702,114 @@ root will not be listable, as it is virtual. Access known paths directly.
 # Group resolve
 
 # Group stats
+Query IPFS statistics.
 
-## bw
+This command can't be called directly.
+
+## bw [POST /stats/bw{&peer,proto,poll,interval}]
+Print ipfs bandwidth information.
+
++ Parameters
+    + peer (string, optional) - Specify a peer to print bandwidth for.
```

Comment:
there is no default value, specifying a peer changes the behaviour of the command to only print bandwidth utilization between our peer and the given peer. Otherwise it shows overall bandwidth.